Packs/CVE_2022_30190/README.md

```diff
@@ -0,0 +1,23 @@
+On **May 27th**, a new Microsoft Office Zero-Day was discovered by Nao_sec. 
```

Comment:
Let's add first
This pack is part of the [Rapid Breach Response (https://xsoar.pan.dev/marketplace/details/MajorBreachesInvestigationandResponse) pack.

Reply:
Done